content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -520,12 +520,6 @@ it(name, config, fn)
 specify(name, config, fn)
 ```
 
-### Test Configuration
```

Comment:
Duplicate of [line 503](https://github.com/cypress-io/cypress-documentation/blob/43bb00bd6ae0348bd4b1ed4dd68f4b05cf3ca688/content/guides/core-concepts/writing-and-organizing-tests.md?plain=1#L503).